frontend/mulighetsrommet-veileder-flate/src/components/filtrering/Filtermeny.tsx

```diff
@@ -18,6 +19,7 @@ const Filtermeny = () => {
         setSokefilter={(search: string) => setFilter({ ...filter, search })}
       />
       <Accordion>
+        <BrukersEnhetFilter />
```

Comment:
Lurer på om vi bare skulle kalt dette for `NavEnhetFilter` (eller noe tilsvarende) i stedet for `BrukersEnhetFilter`. Mest for å signalisere at vi kan filtrere på enheter uavhengig om det er en bruker i kontekst eller ikke.